test/helpers/components/edgeconnect/edgeconnect.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/status"
 	dynatracev1alpha1 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1alpha1"
```

Comment:
```suggestion
    v1alpha1 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1alpha1"
```
Because consistency. It was used like this at another location,

Reply:
fixed in 3f1de959